app/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.test.tsx

```diff
@@ -166,6 +166,15 @@ describe("TableAudience", () => {
     ).not.toBeInTheDocument();
   });
 
+  it("Renders show button for recipe json", () => {
+    const { experiment } = mockExperimentQuery("demo-slug");
+    render(<Subject {...{ experiment, withFullDetails: true }} />);
+    expect(screen.queryByTestId("experiment-recipe-json")).toBeInTheDocument();
+    expect(screen.queryByText("Show More")).toBeInTheDocument();
```

Comment:
It is possible to check here if we're showing a certain number of lines of JSON? (in addition to checking for the "show more" button)

Reply:
also, i don't think it's possible, but I'm gonna ask anyway: is it possible for this json to be empty? 😅 (or less than 2 lines)

Reply:
sure, I will update that test case, regarding question 2- it can't be empty as when we create an experiment we have some mandatory stuff like application, name etc etc.. 
![image](https://user-images.githubusercontent.com/104033388/187489318-834fd87a-2fbc-4fd4-86a4-cc585f00607d.png)